index.js

```diff
@@ -573,10 +573,18 @@ class BitbucketScm extends Scm {
      * @param  {String}   config.token        The token used to authenticate to the SCM
      * @param  {String}   config.url          Target Url of this commit status
      * @param  {String}   [config.jobName]    Optional name of the job that finished
+     * @param  {Number}   [config.pipelineId] Pipeline ID
      * @return {Promise}
      */
     _updateCommitStatus(config) {
+        let context = `Screwdriver/${config.pipelineId}`;
```

Comment:
It looks like `pipelineId` is optional, in that case won't this become `Screwdriver/undefined`?